### DISCUSSION TITLE
new version or  updated easylist?

### DISCUSSION THREAD

**Discussion by jindamvani** [easylist/easylist#15752] [Q&A]
* Right now i have subscribed to pull requests of this repo to know whether list is updated?
* Is there  any way to know?

**Comment by krystian3w** [comment]
Certainly a PR subscription will not give 100% notifications of every change, because not everything goes in PR (people with "write+save" privileges tend to publish something right away and rather ask about the consequences on private Slack instead of in public PR visible to about 7 billion people).

If the "all activity" does not work, then you have to write yourself a listener on the "gh-pages" branch (as there is no longer a ready-made).

I don't recommend "all activity" because why do we need notifications about ads/tracker/annoyances on pages we never go to and won't help cut them out - github itself may suggest after time that EL notifications are overworking us and may be worth revoking; only I'm not sure if they will also send such a notification by e-mail, as someone doesn't like the bell icon from the header (or on a long-lived account didn't activate it, as it wasn't preferred by default and may continue not to be due to the fact that in the e-mail we have immediately comments to Issues/PR/commits (subscribed or pinged by nickname) without viewing it in thread mode on the site to find out what the uproar is about).

Certainly, in the official smartphone app it is possible to enable notifications of successful/unsuccessful actions: https://github.com/easylist/easylist/actions/workflows/pages/pages-build-deployment
(I have recently from my organization that the task aborted actions because there was nothing to do within his.)
